src/test/java/com/newworld/saegil/authentication/service/AuthenticationServiceTest.java

```diff
@@ -188,21 +176,25 @@ class Context_with_invalid_token {
                 // given
                 final User user = new User("권예진", "http://example.com/profile.jpg", "1234567", OAuth2Type.KAKAO);
                 userRepository.save(user);
-                final Token token = tokenProcessor.generateToken(LocalDateTime.now(), new PrivateClaims(user.getId()).toMap());
+                final Token token = tokenProcessor.generateToken(LocalDateTime.now(),
+                        new PrivateClaims(user.getId()).toMap());
                 blacklistTokenRepository.save(new BlacklistToken(user.getId(), TokenType.ACCESS, token.accessToken()));
-                blacklistTokenRepository.save(new BlacklistToken(user.getId(), TokenType.REFRESH, token.refreshToken()));
+                blacklistTokenRepository.save(
+                        new BlacklistToken(user.getId(), TokenType.REFRESH, token.refreshToken()));
```

Comment:
given 절은 비즈니스로직이 아니라 단순한 "데이터 세팅"이기 때문에 "save"의 의미가 더 중요하고, 한 줄에 읽히는 게 중요하다고 생각해서 일부러 개행을 안 한 거긴 합니다. 그래도 만약 개행을 해야 한다면 아래처럼 하는게 더 좋을 것 같아요.
파라미터를 개행하는 경우 괄호의 시작과 끝, 각 파라미터 값의 시작과 끝이 명확한 걸 더 선호합니다.

```suggestion
                blacklistTokenRepository.save(
                        new BlacklistToken(user.getId(), TokenType.REFRESH, token.refreshToken())
                );
```